README.md

```diff
@@ -14,15 +31,19 @@ While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
 
 ## Table of Contents
-* [Example](#example)
+* [Examples](#examples)
 * [SQL Injection Prevention using Parameterization](#sql-injection-prevention-using-parameterization)
 * [Prepared Statements](#prepared-statements)
 * [Schema Management](#schema-management)
 * [Query Examples](#query-examples)
 * [List of plugins](#list-of-plugins)
 * [License](#license)
 
-## Example
+## Examples
+A Swift-Kuery tutorial on adding a PostgreSQL database to a foodTracker app is available [here](https://github.com/IBM/FoodTrackerBackend/blob/master/AddDatabase.md).
```

Comment:
The link here is broken.

Reply:
I don't like the fact that we are listing here a tutorial and sample application before we've even presented our simple example. My suggestion is to rename the title here to Example and move the tutorial and sample application into a new section later in the README under a new title such as Tutorial and Sample Application.